Add landingpage for removed integrations (#16120)
home-assistant/home-assistant.github.io: source/more-info/adr0004.markdown

---
title: "Webscraping"

> description: "With ADR0004 integrations that preform webscraping is not allowed and will be removed."

Reviewer: ```suggestion
description: "With ADR0004 integrations that preform webscraping are not allowed and will be removed."
```

> description: "With ADR0004 integrations that preform webscraping are not allowed and will be removed."

Reviewer 2: ```suggestion
description: "With ADR-0004 integrations that perform web scraping are not allowed and are removed."
```

sidebar: false
---

The integration you requested has been removed because it was doing webscraping.

> This is now allowed. To learn more about that decision, [have a look here][ADR0004].

Reviewer: ```suggestion
This is not allowed. To learn more about that decision, [have a look here][ADR0004].
```

> The integration you requested has been removed because it was doing webscraping.
> This is not allowed. To learn more about that decision, [have a look here][ADR0004].

Reviewer 2: ```suggestion
The integration you requested has been removed because it was doing web scraping.
This is not allowed. To learn more about that decision, [have a look here][ADR-0004].
```

As an alternative you can look at the [`scrape`](/integrations/scrape) integration, browse our list of [other integrations](/integrations/).

> [ADR0004]: https://github.com/home-assistant/architecture/blob/master/adr/0004-webscraping.md

Reviewer 2: ```suggestion
[ADR-0004]: https://github.com/home-assistant/architecture/blob/master/adr/0004-webscraping.md
```
